infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -17,15 +17,53 @@
 
 NPROC=$(nproc)
 
+TARGET_TO_BUILD=
+case $(uname -m) in
+    x86_64)
+      TARGET_TO_BUILD=X86
+      ARCHITECTURE_DEPS="g++-multilib"
+      # Use chromium's clang revision.
+      export CC=$WORK/llvm-stage1/bin/clang
+      export CXX=$WORK/llvm-stage1/bin/clang++
+      ;;
+    aarch64)
+      TARGET_TO_BUILD=AArch64
+      ARCHITECTURE_DEPS="clang lld g++"
+      export CC=clang
```

Comment:
Can you add a comment here on why we use the distro's clang here instead of the chromium one?

Reply:
Done